charts/refarch-template/charts/backend/templates/deployment.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Chart.Name }}
+  labels:
```

Comment:
ich würde auch die Möglichkeit zusätzliche Labels zu erlauben. 

Es wollte jemand auch hier in der Topologie Ansicht ändern, dass geht nur mit speziellen Labels.
 https://docs.openshift.com/container-platform/4.15/applications/odc-viewing-application-composition-using-topology-view.html#odc-labels-and-annotations-used-for-topology-view_viewing-application-composition-using-topology-view